README.md

```diff
@@ -39,12 +39,12 @@ Or, if using `apt-get`, use one of these:
 * [`mono-develop`](https://packages.debian.org/search?keywords=mono-devel) (`mono`)
 * [`mono-complete`](https://packages.debian.org/search?keywords=mono-complete) (`mono`+`xbuild`)
 
-Currently supported versions of Mono: **4.6**, **5.4**
+Currently supported versions of Mono: **4.8**, **5.16**
```

Comment:
The Travis CI build tests with 4.6, and I locally run with 5.4. So those are still tested.

Adding 4.8 and 5.16 is good, but what do they relate to? How are they tested?

Reply:
related with #46 

4.6 doesn't work correctly.